tests/Nervos/SignerTests.cpp

```diff
@@ -0,0 +1,426 @@
+// Copyright © 2017-2021 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "HexCoding.h"
+#include "Nervos/Address.h"
+#include "Nervos/Serialization.h"
+#include "Nervos/Signer.h"
+#include "PrivateKey.h"
+#include "PublicKey.h"
+#include "proto/Nervos.pb.h"
+#include "uint256.h"
+#include "../interface/TWTestUtilities.h"
+#include <TrustWalletCore/TWAnySigner.h>
+
+#include <gtest/gtest.h>
+
+using namespace TW;
+using namespace TW::Nervos;
+
+Proto::SigningInput getInput1() {
+    auto input = Proto::SigningInput();
+    input.set_to_address("ckb1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsqdty"
+                         "q04tvp02wectaumxn0664yw2jd53lqk4mxg3");
+    input.set_change_address("ckb1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xws"
+                             "qds6ed78yze6eyfyvd537z66ur22c9mmrgz82ama");
+    auto amount = Serialization::encodeUint256(uint256_t(10000000000), 16);
```

Comment:
please add some blank lines between code snippets, a bit hard to read them 🤦‍♂️

Reply:
Done